python/revng/api/__init__.py

```diff
@@ -12,6 +11,8 @@
 from .manager import Manager
 from .rank import Rank
 
+__all__ = ["Manager", "Rank", "initialize", "shutdown"]
+
```

Comment:
I guess this is to expose less stuff to users?

Reply:
It does 2 things:
* flake8 will not complain if those symbols are unused
* if you do `from revng.api import *` it will only import those symbols